argo-cd-apps/base/pipeline-service.yaml

```diff
@@ -7,9 +7,10 @@ spec:
   project: default
 
   source:
-    path: components/pipeline-service
-    repoURL: https://github.com/redhat-appstudio/infra-deployments.git
-    targetRevision: main
+    path: operator/gitops/argocd/pipeline-service
+    repoURL: https://github.com/openshift-pipelines/pipeline-service.git
+    targetRevision: new-main
```

Comment:
I would do this in components/pipeline-service -> change kustomization.yaml, that's how other services are managed.